carbonmark/lib/api/swr.keys.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  GetUsersWalletorhandlePathParams,
+  GetUsersWalletorhandleQueryParams,
+} from ".generated/carbonmark-api-sdk/types";
+import { urls } from "lib/constants";
+
+/**
+ * Construct stable URL as key for SWR. When this key changes, data is re-fetched.
+ * Always includes network default 'polygon'
+ */
+export const getUsersWalletorhandleKey = (
```

Comment:
Nit: As @sprrwhwk mentioned in other comments, renaming this and all other associated names to:

```suggestion
export const getUsersWalletOrHandleKey = (
```